### PR TITLE
Clarify governance policy for Peers merging files in the browser subdirectory

### PR DESCRIPTION
**Pull request by sideshowbarker**
This change updates the `GOVERNANCE.md` file to clarify that Peers may merge changes to files in the `browsers` directory — because those are compat data updates, not schema updates. See https://github.com/mdn/browser-compat-data/pull/7211#issuecomment-720027153

### CODE REVIEW

**Comment by vinyldarkscratch** [comment]
Looking good to me!  Perhaps also add a little note in the table at the bottom, something like "Merge data PRs (including browser data)"?

**Comment by sideshowbarker** [comment]
> Perhaps also add a little note in the table at the bottom, something like "Merge data PRs (including browser data)"?

Good point — added now

File: GOVERNANCE.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Peers:
 - Are expected to work on public branches of their forks and submit pull requests to the master branch.
 - Must submit pull requests for all their changes.
 - May label and close issues.
-- May only merge other people's pull requests that relate to compat data updates.
+- May only merge other people's pull requests that relate to compat data updates. (Note: Peers may merge changes to the files in the `browsers` directory — because those are compat data updates, not schema updates).
```
Comment (ddbeck) [inline GOVERNANCE.md:38]: suggestion: split this into two bullets, one about compat data and one about browser data

This also gives me an opportunity to make one fine distinction. Adding or removing browsers is _technically_ a schema change, but it seems to me that we might someday want to de-duplicate that.

```suggestion
- May merge other people's pull requests that relate to compat data updates.
- May merge other people's pull requests that relate to browser data updates (excluding the addition or removal of browsers).
```

File: GOVERNANCE.md [hunk 2]
```diff
@@ -155,7 +155,7 @@ The moderator is responsible for summarizing the discussion of each agenda item
 | Open pull requests or issues                           |                  | •            | •     | •      |
 | Review pull requests or comment on issues              |                  | •            | •     | •      |
 | Label issues and PRs                                   |                  |              | •     | •      |
-| Merge compat data PRs                                  |                  |              | •     | •      |
+| Merge compat data PRs (includes `browser` directory)   |                  |              | •     | •      |
```
Comment (ddbeck) [inline GOVERNANCE.md:158]: suggestion: an extra row, for symmetry with the bullets above

```suggestion
| Merge compat data PRs                                  |                  |              | •     | •      |
| Merge browser data PRs                                  |                  |              | •     | •      |
```

**Review by ddbeck — DISMISSED** [review]
Yes, this seems like a good clarification to make. I've made a couple suggestions to both wordsmith this and to make a clarification about what browser data is. I'd like your thoughts on it!

**Comment by sideshowbarker** [comment]
> I've made a couple suggestions to both wordsmith this and to make a clarification about what browser data is. I'd like your thoughts on it!

Those changes looked great to me — I went ahead and committed them

**Comment by ddbeck** [comment]
@sideshowbarker sorry, my suggestions broke the markdown linting. If you'd run the fix then we can get this merged. 👍

**Comment by sideshowbarker** [comment]
> @sideshowbarker sorry, my suggestions broke the markdown linting. If you'd run the fix then we can get this merged. 👍

Yup — just now pushed a fix for that

**Review by ddbeck — APPROVED** [review]
Thank you! 🎉

**Pull request merged**